biome.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.9.1/schema.json",
+  "files": {
+    "include": ["runtime"],
+    "ignore": ["runtime/zstd.ts"]
+  },
+  "formatter": {
+    "enabled": true,
+    "useEditorconfig": true
+  },
+  "linter": {
+    "enabled": false
+  },
+  "organizeImports": {
+    "enabled": true
```

Comment:
What is this about ?

Reply:
It wasn't really necessary, but the functionality to reorder imports.